ibis/expr/types.py

```diff
@@ -181,7 +181,7 @@ def op(self):
     def _factory(self):
         return type(self)
 
-    def execute(self, limit='default', params=None, **kwargs):
+    def execute(self, limit='default', params=None, client=None, **kwargs):
         """
         If this expression is based on physical tables in a database backend,
```

Comment:
I didn't check this docstring but I maybe it will fail (the summary, 1st line, generally it is just one line). 
could you check using pydocstyle?

maybe we need to fix a current issue on setup.py .. the pydocstyle section should be something like

```
[pydocstyle]
inherit = false
convention = numpy
```

and you can run pydocstyle just for the file you want to check, eg:

`pydocstyle ibis/expr/types.py`

you don't need to fix all the docstring issues .. just for the functions you have touched.